src/sentry/sentry_metrics/consumers/indexer/batch.py

```diff
@@ -440,7 +440,7 @@ def reconstruct_messages(
                     value = old_payload_value["value"]
                     assert isinstance(value, (int, float, list))
                     new_payload_v1: Metric = {
-                        "tags": new_tags,
+                        "tags": cast(dict[str, int], new_tags),
```

Comment:
why is this cast now necessary?

Reply:
No clue as to why, did not change the tags definition